### PR TITLE
Fixes invalid aria-owns/ activedescendant attributes on BasePicker

### LINKED ISSUE #4551: Accessibility- BasePicker aria-owns and activedescendant field should not be set unless suggestions are rendered
#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes

Requested priority: Normal

Products/sites affected: OneDrive share dialog

#### Describe the issue:

BasePicker has aria-owns set to "suggestion-list" even when Suggestions.tsx is not in the DOM. 

#### Expected behavior:

BasePicker aria-owns behavior should mimic aria-expanded behavior.

### If applicable, please provide a codepen repro:

N/A

### PR DESCRIPTION
**Pull request by rebeccaballantyne**
#### Pull request checklist

- [x ] Addresses an existing issue: Fixes #4551
- [x ] Include a change request file using `$ npm run change`

#### Description of changes

Fixes aria-owns issue on BasePicker. Aria-owns should only be set to "suggestion-list" when suggestions are rendered.

#### Focus areas to test

N/A

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx [hunk 1]
```diff
@@ -205,7 +205,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
                 onInputValueChange={ this.onInputChange }
                 suggestedDisplayValue={ suggestedDisplayValue }
                 aria-activedescendant={ activeDescendant }
-                aria-owns='suggestion-list'
+                aria-owns={ !!this.state.suggestionsVisible ? 'suggestion-list' : '' }
```
Comment (manishgarg1) [inline packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx]: >this.state.suggestionsVisible [](start = 30, length = 29)

I don;t think you really need to use the !! prefix

Comment (rebeccaballantyne) [reply packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx]: True, because it's a ternary. Removing.

**Comment by manishgarg1** [comment]
@joschect, are you good with this?

**Comment by KevinTCoughlin** [comment]
Change overall LGTM, but one minor nit given the spec https://www.w3.org/WAI/PF/aria-1.1/states_and_properties#aria-owns

>The value of the aria-owns attribute is a space-separated list of IDREFS that reference one or more elements in the document by ID. The reason for adding aria-owns is to expose a parent/child contextual relationship to assistive technologies that is otherwise impossible to infer from the DOM.

So an empty string for the value when the suggestion list is not shown may be invalid. Though my [WAVE extension](http://wave.webaim.org/extension/) doesn't seem to flag it when testing.

Might be better to return `null` or `undefined`? Thoughts?

File: packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx [hunk 2]
```diff
@@ -205,7 +205,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
                 onInputValueChange={ this.onInputChange }
                 suggestedDisplayValue={ suggestedDisplayValue }
                 aria-activedescendant={ activeDescendant }
-                aria-owns='suggestion-list'
+                aria-owns={ this.state.suggestionsVisible ? 'suggestion-list' : '' }
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx]: Should we use `undefined` rather than blank string?

**Comment by dzearing** [comment]
Yes `undefined`. The `null` value is not liked by the latest react typings.

File: packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx (hunk 2 above)
Comment (rebeccaballantyne) [reply packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx]: Thanks. Will update!

**Pull request merged**